_data/AniruddhaTapas/AniruddhaTapas.md

```diff
@@ -0,0 +1,7 @@
+---
+name: Aniruddha Tapas
+institution: Northeastern University
+profile_pic: aniruddha.png
```

Comment:
image path name seems incorrect, please change it from `aniruddha.png` to `aniruddha.jpg`

Reply:
Thanks for letting me know! I have fixed this in my recent commit. Please take a look.